project_risk/data/project_risk_category_data.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
```

Comment:
I'd suggest create a `<data noupdate="1">` section in order to prevent risk category description changes overwrite when updating the module